OpenNMT/onmt/Models.py

```diff
@@ -29,10 +29,9 @@ def __init__(self, opt, dicts):
             self.word_lut.weight.copy_(pretrained)
 
     def forward(self, input, hidden=None):
-        emb = self.word_lut(input)
-
+        batch_size = input.size(0) # batch first for multi-gpu compatibility
+        emb = self.word_lut(input).transpose(0, 1)
```

Comment:
Looks like the transposes haven't decreased performance (although in a later comment you said it's 300 words/s slower, so I'm not clear). That's good.